### PR TITLE
Add note for apt installations

### PR DESCRIPTION
**Pull request by Tostti**
## Description
This PR adds a note to the upgrade central components section, for apt installations noting that if the `opensearch_dashboards.yml` was modified, the system will ask to either keep the modified version or replace it with the updated one, meaning that the user will have to manually add the changes later. 

image

### Docs building
- [x] Compiles without warnings.
### Code formatting and web optimization
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [x] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

**Review by Desvelao — DISMISSED** [review]
LGTM!

File: source/upgrade-guide/upgrading-central-components.rst
Comment (javimed) [inline source/upgrade-guide/upgrading-central-components.rst:1]: We can place the note before the command. We can expand it as follows


```
         .. note::

            Configuration options might differ across versions. Running the command below prompts you to choose between two options if you modify ``/etc/opensearch_dashboards.yml``:
            
            -  Replace the file with the updated version.
            -  Keep the modified file.
            
            To ensure a smooth update, follow these steps.
            
            #. Backup the file.
            #. Run the command below, choosing to replace the file with the updated version.
            #. Manually reapply any changes.

```

**Review by javimed — CHANGES_REQUESTED** [review]

**Review by javimed — APPROVED** [review]

**Comment by javimed** [comment]
Added a commit with a new suggestion. What about the yum upgrade? Should we remove the file before upgrading to force updating the file?

**Comment by Tostti** [comment]
> Added a commit with a new suggestion which I can easily revert. 
LGTM

>And what about the yum upgrade? Should we remove the file before upgrading to force updating the file?
I think that it should. In that case we'll be sure that the file is completely replaced

**Review by javimed — DISMISSED** [review]

**Review by javimed — APPROVED** [review]

**Review by asteriscos — APPROVED** [review]
LGTM :heavy_check_mark: 

image

**Comment by javimed** [comment]
@Tostti and @Desvelao have agreed to merge these changes.

**Pull request merged**